google-assistant-sdk/CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+0.3.1
+--------
+- Bump dependencies
```

Comment:
Can you explain in one line what is the reason you bumped it? (else it may not be obvious for external developers)